menu.html

```diff
@@ -71,11 +75,12 @@
 */
 </script>
 <div id="menu">
-	<h1>CS:GO HUD</h1>
-	<p> Choose HUD</p>
+	<img style="width: 100%" src="banner.png" alt="CS:GO HUD">
+	<p style="font-family:csfont;"> CHOOSE HUD</p>
 	<span class="controls">CTRL+ALT+Z</span>
-	<p> Minimize HUD</p>
+	<p style="font-family:csfont;"> MINIMIZE HUD</p>
 	<span class="controls">CTRL+ALT+X</span>
-	<p> Close HUD</p>
+	<p style="font-family:csfont;"> CLOSE HUD</p>
 	<span class="controls">CTRL+ALT+C</span>
+	<p>To enable debug mode type "dev = true" in console.</p>
```

Comment:
Could you remove this line or add some script to show this line?

Should only be shown if we are launched via `npm test` or `npm start` and not in the build version.